src/containers/user-profile/profile-info/ProfileInfo.tsx

```diff
@@ -132,6 +137,31 @@ const ProfileInfo = ({ userData, myRole }: ProfileInfoProps) => {
       }
   ].filter(Boolean)
 
+  const {
+    response: listOfChats,
+    loading: isChatsLoading,
+    fetchData
+  } = useAxios<ChatResponse[]>({
+    service: () => chatService.getChats(),
+    defaultResponse: []
+  })
+
+  const onSendMessageClick = () => {
+    const existedChat = listOfChats.find((chat: ChatResponse) => {
```

Comment:
Chat already typed automatically here

```suggestion
    const existedChat = listOfChats.find((chat) => {
```